website/content/docs/hcp/index.mdx

```diff
@@ -0,0 +1,46 @@
+
+The HCP Packer registry stores metadata about your images, including when they were created, where the image exists in the cloud, and what (if any) git commit is associated with your image build. You can use the registry to track information about the golden images your Packer builds produce, clearly designate which images are appropriate for test and production environments, and query for the right golden images to use in both Packer and Terraform configurations.
+
+You can use HCP Packer with both JSON and HCL2 templates. If you are using JSON templates we recommend getting started with 
+the [HCP Packer environment variables](#hcp-packer-environment-variables) and then migrating to HCL when possible. 
+
+This page summarizes the methods you can use to connect JSON and HCL2 templates to the HCP Packer registry. It also provides a full list of HCP Packer environment variables. Refer to the [HCP Packer Template Configuration](https://cloud.hashicorp.com/docs/packer/store-image-metadata/template-configuration) page in the HCP Packer documentation for full configuration details and examples.
+
+### HCP Packer Environment Variables
+
+The following environment variables let you configure Packer to push image metadata to an active registry without changing your template. You can use environment variables with both JSON and HCL2 templates. Refer to [Basic Configuration With Environment Variables](https://cloud.hashicorp.com/docs/packer/store-image-metadata/template-configuration#basic-configuration-with-environment-variables) in the HCP Packer documentation for complete instructions and examples. 
+
+You must set the following environment variables to enable Packer to push metadata to a registry.
+
+- `HCP_CLIENT_ID`         - The HCP client ID of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_CLIENT_SECRET`    - The HCP client secret of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_PACKER_BUCKET_NAME` -  The name of the Image bucket where Packer will store information about each artifact from a build within the configured template. 
+
+
+You can set these additional environment variables to control how metadata is pushed to the registry.
+
+- `HCP_PACKER_BUILD_FINGERPRINT` - A unique identifier assigned to a build run to determine if metadata for a template build on the registry is complete. By default, the HEAD Git SHA 
+for the template file is used to create a fingerprint. If the template is not version controlled, this environment variable must be set to a unique value before executing a build on the template. 
+
+- `HCP_PACKER_REGISTRY`    -  Allows users to toggle HCP Packer publishing off. Allowed values are [0|OFF].
```

Comment:
okay a couple suggestions here for active voice, which makes it easier to read (HCP Packer or the user is doing an action, rather than "must be set"). Also, i tried a new thing for the second one. I wanted to be like "yeah, even if you already configured your template, you can easily stop packer from pushing metadata if you want to."

Question for you --> IF folks are using an HCP Packer Build block in their template, does this still stop Packer from pushing metadata to the registry? Or does this only work when you have a purely environment-variable driven connection?
```suggestion
- `HCP_PACKER_BUILD_FINGERPRINT` - A unique identifier assigned to each build. HCP Packer uses this identifier to determine if metadata for a build on the registry is complete. By default, HCP Packer uses the HEAD Git SHA for the template file. If the template is not version controlled, you must set this environment variable to a unique value before running `packer build`. 

- `HCP_PACKER_REGISTRY`    -  When set, Packer does not push image metadata to HCP Packer from an otherwise configured template. Allowed values are [0|OFF].
```